Up2dateService/wrapperdll/vcpkg.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I guess this file should not be under source control as it is the result of copy during the build process.

Reply:
fixed